announcements/2020-Week-37/apps.md

```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Releases (September 10, 2020):zap:
+
+**Giant Swarm Catalog (Managed Apps)**
+
+1. [Kong app v0.9.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#090---2020-08-25) updates default Kong version to v2.1.3, other dependencies, and syncs with upstream chart v1.8.0
+3. [Kong app v0.9.1](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#091---2020-09-07) splits image repository value to allow switching registry i.e. instead of default Quay, use Alibaba by default in China.
+3. [NGINX IC app v1.9.2](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#192---2020-09-02) upgrades ingress-nginx from v0.34.1 to v0.35.0. Please check the [upstream changelog](https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md#0350) for details on all changes.
+4. [EFK Stack app v0.3.0](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#030---2020-09-04) upgrades to Open Distro v1.9.0.
```

Comment:
```suggestion
1. [Strimzi Kafka Operator app v0.1.0](https://github.com/giantswarm/strimzi-kafka-operator-app/blob/master/CHANGELOG.md#010---2020-08-14) upgrades to upstream from v0.18.0 to v0.19.0. (app can be installed from the Playground Catalog)
2. [Kong app v0.9.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#090---2020-08-25) Update default Kong version to 2.1.3, update other dependencies, sync with upstream chart 1.8.0
3. [Kong app v0.9.1](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#091---2020-09-07) Split image repository value to allow switching registry i.e. instead of default Quay use Alibaba by default in China.
4. [Jaeger Operator app v0.2.0](https://github.com/giantswarm/jaeger-operator-app/blob/master/CHANGELOG.md#020---2020-09-01) Upgrades jaeger-operator version to v1.18.0. Please check the [upstream changelog](https://github.com/jaegertracing/jaeger-operator/blob/master/CHANGELOG.md#1180-2020-05-15) for details on all changes. (app can be installed from the Playground Catalog)
5. [NGINX IC app v1.9.2](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#192---2020-09-02) Upgrades ingress-nginx from v0.34.1 to v0.35.0. Please check the [upstream changelog](https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md#0350) for details on all changes.
6. [EFK Stack app v0.3.0](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#030---2020-09-04) upgrades to Open Distro v1.9.0.
```